### PR TITLE
Allow customisation of the OSD stats screen

### PR DESCRIPTION
**Pull request by DanNixon**
To go with https://github.com/betaflight/betaflight/pull/3133

### CODE REVIEW

File: tabs/osd.js [hunk 1]
```diff
@@ -591,12 +653,22 @@ OSD.msp = {
     }
     return result;
   },
-  encode: function(display_item) {
+  encodeLayout: function(display_item) {
     var buffer = [];
+    if (semver.gte(CONFIG.apiVersion, "1.36.0")) {
```
Comment (mikeller) [inline tabs/osd.js]: This should really go to the end of the message, to keep compatibility with the existing format. See also comments on https://github.com/betaflight/betaflight/pull/3133

**Comment by DanNixon** [comment]
@mikeller Should this be merged now that https://github.com/betaflight/betaflight/pull/3133 has been?

**Comment by mikeller** [comment]
I wasn't aware that this had been merged. It is still breaking compatibility with the existing API, so I think it should be fixed before the change is merged into the configurator.

**Comment by DanNixon** [comment]
There should be no issues with API compatibility.

I've tested this configurator branch with Betaflight 3.1.7 and configurator version 3.1.1 with Betaflight master and both work correctly.

**Comment by mikeller** [comment]
Sorry, you are right. This seems to be good now. Having the MSP for OSD sprinkled in with the rest of the OSD code makes the MSP changes hard to read. :-(

**Pull request merged**